### DISCUSSION TITLE
Image component doesn't accept children

### DISCUSSION THREAD

**Discussion by Jordan1022** [express-labs/pure-react-carousel#393] [Q&A]
The documentation states that the Image component requires children but I cannot seem to get it working without crashing because it's a void tag. I've tried as a prop on the carousel 
`children={<h2>hello</h2>}` and 
```
<Image>
    <h2>hello</h2>
</Image>
```
Any thoughts?

**Comment by Conor-Hinchee** [comment]
Hey, @Jordan1022 thanks for opening this discussion! I am hoping to get some more details from you because I think there might be some other factors that are causing the issue. Would mind sharing a code sandbox example?

This seems to be working for me https://codesandbox.io/s/loving-ritchie-49pzq?file=/src/App.js

**Reply by Jordan1022** [reply]
I realize this doesn't give all the context because I don't have the data coming in etc. but here's my context: 
And I have tried using children prop on the image as well as nested the h2 and h3 inside of the Image
https://codesandbox.io/s/muddy-silence-sv66r?file=/src/App.js

**Reply by Conor-Hinchee** [reply]
I see the `h2` tag being rendered. Did this resolve your problem?  or are you trying to get the text over the image?
